app/consumers/https_consumer.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import random
+import signal
+import time
+from typing import Any, Callable
+
+from consumers.base_consumer import BaseConsumer
+from core.config import settings
+from port_client import ack_runs, claim_pending_runs
+
+logging.basicConfig(level=settings.LOG_LEVEL)
+logger = logging.getLogger(__name__)
+
+
+class HttpsConsumer(BaseConsumer):
+    def __init__(self, msg_process: Callable[[dict], None]) -> None:
+        self.running = False
+        self.msg_process = msg_process
+        self.backoff_seconds = 0
+        self.max_backoff = settings.HTTPS_MAX_BACKOFF_SECONDS
+
+        signal.signal(signal.SIGINT, self.exit_gracefully)
+        signal.signal(signal.SIGTERM, self.exit_gracefully)
+
+    def _exponential_backoff(self) -> None:
+        if self.backoff_seconds == 0:
+            self.backoff_seconds = 1
+        else:
+            self.backoff_seconds = min(self.backoff_seconds * 2, self.max_backoff)
+
+        jitter = random.uniform(0, self.backoff_seconds * 0.1)
+        sleep_time = self.backoff_seconds + jitter
+
+        time.sleep(sleep_time)
+
+    def _reset_backoff(self) -> None:
+        if self.backoff_seconds > 0:
+            self.backoff_seconds = 0
+
+    def start(self) -> None:
+        self.running = True
+
+        while self.running:
+            try:
+                runs = claim_pending_runs(limit=settings.HTTPS_RUNS_BATCH_SIZE)
+                self._reset_backoff()
+                if runs:
+                    logger.info("Claimed %d pending runs", len(runs))
+
+                    for run in runs:
+                        run_id = run.get("_id") or run.get("id")
+
+                        try:
+                            acked_count = ack_runs([run_id])
+                            if acked_count == 0:
+                                logger.warning("Failed to ack run %s", run_id)
+                                continue
+                            logger.info("acked run %s", run_id)
```

Comment:
Align casing with other logs 😅 OCD

```suggestion
                            logger.info("Acked run %s", run_id)
```